src/aus.h

```diff
@@ -0,0 +1,329 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "types.h"
+
+namespace librealsense
+{
+    
+    class aus_value
+    {
+    public:
+        virtual long get() = 0;
+        virtual void set(long value) = 0;
+        virtual void increment() = 0;
+        virtual void decrement() = 0;
+        virtual void start() = 0;
+        virtual void stop() = 0;
+    };
+
+    class aus_timer : public aus_value {
+    
+    public:
```

Comment:
Operations are not atomic.
Meaning it is not thread safe. we need to think about it.

Reply:
aus_timer and aus_counter fixed